src/main/xerus/monstercat/api/response/Track.kt

```diff
@@ -53,16 +50,6 @@ open class Track: MusicItem() {
 		if(titleClean.isNotEmpty())
 			return this
 		
-		if(::release.isInitialized) {
```

Comment:
Even if we were to remove this, it does not belong into this PR. This is supposed to work, so I'm gonna fix it.

Reply:
Merged your implementation in b6a24c5a38d14eba3594c76b12502c2d720a1724